tiledb/sm/array_schema/array_schema_operations.cc

```diff
@@ -227,4 +231,61 @@ void store_array_schema(
   }
 }
 
+shared_ptr<ArraySchema> load_array_schema(
+    const Context& ctx, const URI& uri, const Config& config) {
+  // Check array name
+  if (uri.is_invalid()) {
+    throw std::runtime_error("Failed to load array schema; Invalid array URI");
+  }
+
+  if (uri.is_tiledb()) {
+    auto& rest_client = ctx.rest_client();
+    auto&& [st, array_schema_response] =
+        rest_client.get_array_schema_from_rest(uri);
+    throw_if_not_ok(st);
+    return std::move(array_schema_response.value());
```

Comment:
```suggestion
    return std::move(array_schema_response).value();
```

Nit: This will have the same behavior and use the rvalue reference-qualified overload of `value()`.

Reply:
Done, nice!